txtl-liposome_thin-film.md

```diff
@@ -0,0 +1,109 @@
+rhodamine membrane dye to be hydrated later to prepare lipid
+vesicles.  This procedure works best for studies with bulk small
+unilamellar vesicles, but can be used to prepare giant (>1 um)
+vesicles (just skip the extrusion step).
+
+### Materials
+
+A. Glass vials (any will do, but we use [2mL Fisherbrand Class B Clear
+   Glass Threaded vials: cat #:
+   03-339-21A](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738)
+
+B. Glass syringes of various sizes (ex. Hamilton gastight cat #:
+   14-815-238, but any will do) for use with lipids and chloroform
+
+C. Phosphate buffered saline 290 mOsm (Sigma, P4417-100TAB)
+
+D. Mini Extruder (Avanti Polar Lipids) 
+
+![Materials for TX-TL liposome protocol](txtl-liposome_thin-film-materials.png)
+
+## Method:
+
+Final reaction conditions:
+
+| Reagent | Source | Final working concentration |
+| ------- | ------ | --------------------------- |
+| DOPC    | Avanti 850375C | 2.5 E -5 mols/vial (1 mL of hydration yields 20 mM vesicles)* |
+| Rhodamine Lissamine PE | Avanti 810150 C | 0.1 mol % of POPC |
+| Cholesterol | Santa Cruz Biotech sc-202539 | 2X molarity of DOPC, if used |
+
+\* For microscopy, lower concentration of vesicles hydrated in low salt solutions yields higher quality vesicles for imaging (recommend you make 125 uM vesicles once hydrated)
+
+### Step 1: Prepare DOPC films (2.5E-4 mols DOPC/vial)
+
+1. Determine the volume and molarity DOPC vesicles you want to create
+   ex: 1 mL of 20 mM DOPC vesicles = 2.5E-5 mols of DOPC. The stock of
+   DOPC is 25 mg/mL. To make giant, nice looking vesicles, it’s best
+   to prepare a more dilute sample of vesicles (like 200 uM). For
+   small unilamellar vesicles, you can work with much higher
+   concentrations of lipids (we can go up to 130 mM for DOPC)
+
+2. Determine the dye and molar % of the dye : DOPC you would like
+   ex. for 0.1 mol% rhodamine DHPE :
+
+3. 2.5e-6 moles DOPC×(0.1 moles Rhod Liss PE)/(100 moles
+   DOPC)×L/(moles Rhod in stock)=volume Rhod Liss PE stock to add
+
+4. Add 2e-5 mols DOPC and appropriate amount of Rhodamine PE. Transfer
+   with a Hamilton (glass) syringe into a glass vial since you are
+   working with organic solvents. Cap and vortex briefly to ensure the
+   two components are well mixed.
+
+5. Remove cap and allow the chloroform to evaporate away in the hood.
+
+6. Place the vial into a vacuum chamber for > 1 hrs or leave overnight
+   in order to remove all residual traces of solvent. *Removal of
+   residual chloroform from the lipid film is very important for high
+   quality vesicles
+
+7. Cap and store vials in freezer until ready to be used and hydrate
+   with 1 mL of aqueous hydration buffer when you are ready to prepare
+   vesicles.
+
+### Step 2: Hydrate films and extrude vesicles
+
+1. Add 1 mL of PBS to each vial, place in 60°C oven for 1 hr, then
```

Comment:
Change to degC (to avoid non text characters, which may not always render properly).